### PR TITLE
feat(policies): sort, sorted-by flags

### PR DESCRIPTION
**Pull request by mariiatuzovska**
Add sort, sorted-by flags to policy search command.

### CODE REVIEW

File: .changes/unreleased/🎉 Feature-20230116-184305.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+kind: "\U0001F389 Feature"
+body: Add sort, sorted-by flags to policy search command.
```
Comment (sheldonhull) [inline .changes/unreleased/🎉 Feature-20230116-184305.yaml]: This will go into customer release notes. Can you rephrase to better explain to a potential user and for our changelog what this will provide? They won't have the context of the PR so need to explain it. This message will normally be different than a commit message or PR body. 

Example: the policy search command now has the ability to return presorted data. The sorting can be customized by any available field. 


Hope that helps!

File: .changes/unreleased/🎉 Feature-20230116-184305.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+time: 2023-01-16T18:43:05.47579277+02:00
+custom:
+  azure-boards-workitemid-fixed: '481928'
```
Comment (sheldonhull) [inline .changes/unreleased/🎉 Feature-20230116-184305.yaml:5]: The fixed value means it will automatically change this work item state in the backlog.

Just wanted you to be aware it will close out the PBI or the task.

**Review by sheldonhull — CHANGES_REQUESTED** [review]
@mariiatuzovska  minor improvement to the changelog entry and we're good.

**Review by andrii-zakurenyi — APPROVED** [review]

**Review by sheldonhull — APPROVED** [review]
